decidim-proposals/spec/commands/decidim/proposals/destroy_proposal_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Proposals
+    describe DestroyProposal do
+      describe "call" do
+        let(:component) { create(:proposal_component) }
+        let(:organization) { component.organization }
+        let(:current_user) { create(:user, organization: organization) }
+        let(:other_user) { create(:user, organization: organization) }
+        let!(:proposal) { create :proposal, component: component, author: current_user }
+        let(:proposal_draft) { create(:proposal, :draft, component: component, author: current_user) }
+        let!(:proposal_draft_other) { create :proposal, component: component, author: other_user }
```

Comment:
Can you check the proposal is actually destroyed? I usually do this:


```ruby
described_class.call(proposal, current_user)

expect { proposal.reload }.to raise_error(ActiveRecord::RecordNotFound)
```

Reply:
Yes I'll add it